sources/Context.cpp

```diff
@@ -329,7 +329,12 @@ void Context::execute() {
     // For a power flow calculation it is ok to directly run here a single simulation
     auto simu = boost::make_shared<DYN::Simulation>(jobEntry_, simu_context, networkManager_.dataInterface());
     simu->init();
-    simu->simulate();
+    try {
+      simu->simulate();
```

Comment:
Some error type are missing. It should be similar to the equivalent section in DYNSImulationLauncher.cpp